bigday/templates/base.html

```diff
@@ -8,7 +8,7 @@
     <meta name="description" content="">
     <meta name="author" content="">
     <!-- Modified from https://github.com/IronSummitMedia/startbootstrap-creative -->
-    <title>Cory and Rowena's big day</title>
+    <title>{{ support_email }}'s big day</title>
```

Comment:
Should this not be `{{ couple_name }}`?

Reply:
Yes.  I fixed it in the other PR this morning.